ls1intum/Artemis
# `General`: Fix responsiveness in user list view

### Checklist
#### General


#### Client

### Motivation and Context



There are some responsiveness issues in the header of a course user list view:
![grafik](https://user-images.githubusercontent.com/23171488/182661062-4db9315e-6113-4434-9218-745c3e21a8ca.png)

### Description


Fixed those issues by replacing some weird float styles with proper flexboxing

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Log in to Artemis
2. Navigate to a Course Page
3. Scroll down, click the name of the student group (or instructors, tutors ...)
4. In the list view, resize your viewport horizontally like in the video and make sure that everything looks good and is responsive

### Review Progress





#### Code Review
#### Manual Tests

### Screenshots


![ezgif-3-15bbb95a5b](https://user-images.githubusercontent.com/23171488/182661214-f2c872ed-c48e-4a3d-a85a-3909fc8a63f0.gif)
